### PR TITLE
Fixes to make files for lua exports

### PR DESCRIPTION
**Pull request by projectkk2glider**

### CODE REVIEW

File: companion/src/firmwares/opentx/simulator/CMakeLists.txt [hunk 1]
```diff
@@ -20,10 +20,18 @@ add_custom_target(commondeps
...
-)
+macro(add_lua_export_target target)
+  add_custom_command(
+    OUTPUT ${RADIO_SRC_DIRECTORY}/lua/lua_exports_${target}.inc
+    COMMAND ${CMAKE_C_COMPILER} -E ${ARGN} -DEXPORT ${INTTYPES_INCLUDE_OPTION} myeeprom.h | grep LEXP > lua_exports_${target}.txt
+    COMMAND python ../util/luaexport.py ${C9X_VERSION} lua_exports_${target}.txt lua/lua_exports_${target}.inc lua_fields_${target}.txt
+    WORKING_DIRECTORY ${RADIO_SRC_DIRECTORY}
+  )
+  add_custom_target(lua_export_${target} DEPENDS ${RADIO_SRC_DIRECTORY}/lua/lua_exports_${target}.inc)
+endmacro(add_lua_export_target)
```
Comment (projectkk2glider) [inline companion/src/firmwares/opentx/simulator/CMakeLists.txt]: This part was ported from next branch

File: jenkins/nightly-21/release-companion.sh [hunk 2]
```diff
@@ -14,7 +14,13 @@ source ./version.sh
 
 DESTDIR=/var/www/html/downloads-${version}/nightly/companion
 
+# clean radio source
+cd opentx/radio/src
+make clean
+
```
Comment (projectkk2glider) [inline jenkins/nightly-21/release-companion.sh]: I think it is important to clean the radio dir before the new Companion is built

File: jenkins/nightly-21/release-companion.sh [hunk 3]
```diff
@@ -14,7 +14,13 @@ source ./version.sh
...
 # create companion rpm
+rm -rf $DIR/companion-build/
+mkdir -p $DIR/companion-build/
```
Comment (projectkk2glider) [inline jenkins/nightly-21/release-companion.sh]: Also it is better to start with the empty Companion build directory

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**